src/components/ParentNavigationSubtitle.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import CONST from '../CONST';
+import Text from './Text';
+import reportPropTypes from '../pages/reportPropTypes';
+import PressableWithoutFeedback from './Pressable/PressableWithoutFeedback';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import useLocalize from '../hooks/useLocalize';
+
+const propTypes = {
+    parentNavigationSubtitleData: PropTypes.shape({
+        // Title of root report room
+        rootReportName: PropTypes.string,
+
+        // Name of workspace, if any
+        workspaceName: PropTypes.string,
+    }).isRequired,
+
+    /** report */
+    report: reportPropTypes,
```

Comment:
We don't seem to really need the whole report here. Let's just use `parentReportID`